Training/Program.cs

```diff
@@ -8,21 +8,44 @@
    else
       Console.WriteLine ("Enter a valid integer");
 }
-static long NthArmstrongNumber (int nArmsNum) {
-   long[] result = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 153, 371, 407, 1634, 8208, 9474, 54748, 92727, 93084, 548834, 1741725, 4210818, 9800817, 9926315, 24678051 };
-   return result[nArmsNum - 1];
-}
+int nthArmsNum = CalculateNthArmstrong (nArmsNum);
 switch (nArmsNum) {

```

Comment:
Please change the logic, because Armstrong number of 0 to 9 is same only.
Instead of Console.Write use Console.WriteLine